CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Fix client creation with proxy (#951)
+- Make `attach_stacktrace` not attach a stack trace when capturing an Exception in order to prevent sentry.io from showing a useless backtrace (#940, #957)
```

Comment:
Since this issue is not related to [sentry.io](https://sentry.io) but to the server in general, can you please update the description to ``Fix sending of both `event.stacktrace` and `event.exceptions` when `attach_stacktrace = true``. Also please drop the reference to the issue and leave only the PR as that is enough

Reply:
fixed